recipes/libcap/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+cmake_minimum_required(VERSION 3.1)
+project(PackageTest CXX)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+add_executable(example example.cpp)
```

Comment:
I would make the example a c source instead of cpp.
This tests whether the library is really c only (which it is).
But just in case...

Reply:
I don't mind pure C test package.